### PR TITLE
Fix for #6595: Background music stopped on iOS game startup

### PR DESCRIPTION
**Pull request by dtaylorus**
Removed the code in OpenALSoundController that was setting AVAudioSessionCategory to SoloAmbient and overriding the game developer setting.

### CODE REVIEW

File: MonoGame.Framework/Audio/OpenALSoundController.cs [hunk 1]
```diff
@@ -237,15 +237,7 @@ private bool OpenSoundController()
...
-                // This is the default audio session category on iOS.
-                //
-                //   Your audio is silenced by screen locking and by the Silent switch (called the Ring/Silent switch on iPhone).
-                //
-                //   By default, using this category implies that your app�s audio is nonmixable�activating your session will
-                //   interrupt any other audio sessions which are also nonmixable.
-                //
-                //
-                AVAudioSession.SharedInstance().SetCategory(AVAudioSessionCategory.SoloAmbient);
+                // NOTE: Do not override AVAudioSessionCategory set by the game developer per #6595.
```
Comment (Jjagg) [inline MonoGame.Framework/Audio/OpenALSoundController.cs]: You don't need to add this comment.

Comment (dtaylorus) [reply MonoGame.Framework/Audio/OpenALSoundController.cs]: The comment is intended to reduce the chance that someone adds that code again, which seems to be happening over the history of the project.

Comment (Jjagg) [reply MonoGame.Framework/Audio/OpenALSoundController.cs]: Changed it to a URL, I'll leave it in.

**Comment by Jjagg** [comment]
Thanks @dtaylorus :)

**Comment by dtaylorus** [comment]
Awesome, thank you @Jjagg !

**Pull request merged**